appimagecraft.yml

```diff
@@ -0,0 +1,34 @@
+version: 1
+
+project:
+  name: org.squidowl.halloy
+  # version: 0.1.0
+  version_command: awk -F'[ ="]+' '$1 == "version" { print $2;exit;}' "$PROJECT_ROOT"/Cargo.toml
+
+build:
+  script:
+    commands:
+      - cargo build --release
+
+scripts:
+  post_build:
+    - mkdir -p "$BUILD_DIR"/AppDir/usr/share/metainfo
+    - cp "$PROJECT_ROOT"/assets/linux/org.squidowl.halloy.appdata.xml "$BUILD_DIR"/AppDir/usr/share/metainfo/
+    - |2
+      cat > "$BUILD_DIR"/halloy.desktop <<EOF
```

Comment:
I suspect the reason the desktop file and meta file weren't working in conjunction is because the desktop file should be named `org.squidowl.halloy.desktop` since our meta file has `  <launchable type="desktop-id">org.squidowl.halloy.desktop</launchable>` set.

I noticed in your custom meta file, you changed it to be ` <launchable type="desktop-id">halloy.desktop</launchable>`.

If we use the original `.desktop` and `appstream.xml` files, but name this `org.squidowl.halloy.desktop` here instead of `halloy.desktop`, does it work?

Reply:
It's also possible we need to put it under `$BUILD_DIR"/AppDir/usr/share/applications/org.squidowl.halloy.desktop`